src/Executor/ReferenceExecutor.php

```diff
@@ -927,28 +928,27 @@ function ($previous, $value) use ($callback) {
     }
 
     /**
-     * Complete a list value by completing each item in the list with the
-     * inner type
+     * Complete a list value by completing each item in the list with the inner type.
      *
-     * @param FieldNode[] $fieldNodes
-     * @param mixed[]     $path
-     * @param mixed       $result
+     * @param FieldNode[]         $fieldNodes
+     * @param mixed[]             $path
+     * @param mixed[]|Traversable &$results
```

Comment:
is specifying `&` in phpdocs standard thing? I'm not against just don't remember seeing it anywhere else 🤔

Reply:
It is not in the standard. Removed